L1Trigger/L1THGCal/src/HGCalTriggerTools.cc

```diff
@@ -90,6 +100,7 @@ unsigned HGCalTriggerTools::layers(DetId::Detector type) const {
     case DetId::HGCalHSc:
       layers = bhLayers_;
       break;
+    // FIXME: to do HFNose
```

Comment:
should this be fixed in this PR?

Reply:
In HGCalTriggerTools::layers(DetId::Detector type) I do not have access to the subdet so cannot set up the switch to the HFNose.

Downstream the code the HGCalTriggerTools::layers(ForwardSubdetector type) is in use.
See for example the changes in https://github.com/cms-sw/cmssw/pull/28461/files#diff-ffd813bde161fec49fe1259f558ea728R91

Reply:
This function cannot handle HFNose. 
At some point we should think if there is a better way to handle all cases, maybe merging the different `layers` functions. But it will require to check everywhere the functions are called. I propose to do this in an other PR. We can keep the `FIXME` in the code to remember that.